storage/server/frontend.go

```diff
@@ -240,12 +240,42 @@ func (s *server) NVMeNamespaceStats(ctx context.Context, in *pb.NVMeNamespaceSta
 //////////////////////////////////////////////////////////
 
 func (s *server) VirtioBlkCreate(ctx context.Context, in *pb.VirtioBlkCreateRequest) (*pb.VirtioBlkCreateResponse, error) {
-	log.Printf("Received from client: %v", in)
+	log.Printf("VirtioBlkCreate: Received from client: %v", in)
+	params := struct {
```

Comment:
At some point you may want to define these structures somewhere, but fine for now.

Reply:
YES!